index.js

```diff
@@ -43,7 +43,7 @@ menu[menu.length - 1].submenu.push({
   }
 })
 
-app.on('ready', () => {
+const onReady = () => {
```

Comment:
Minor nit: could we make this a normal function if possible? I'm cool if you like arrow functions but I always find these `var foo = function() {}` style functions super hard to read